res/xml/number_row.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <row height="0.75">
-    <key key0="1"/>
-    <key key0="2"/>
-    <key key0="3"/>
-    <key key0="4"/>
-    <key key0="5"/>
-    <key key0="6"/>
-    <key key0="7"/>
-    <key key0="8"/>
-    <key key0="9"/>
-    <key key0="0"/>
+    <key key0="1" sw="!"/>
+    <key key0="2" sw="@"/>
+    <key key0="3" sw="#"/>
+    <key key0="4" sw="$"/>
+    <key key0="5" sw="%"/>
+    <key key0="6" sw="^"/>
+    <key key0="7" sw="&amp;"/>
+    <key key0="8" sw="*"/>
+    <key key0="9" sw="("/>
+    <key key0="0" sw=")"/>
```

Comment:
I think the parentheses on separate keys is fine. These keys are shorter and they shouldn't be made more crammed.